scripts/tests/test-ci-all.sh

```diff
@@ -26,6 +26,13 @@ cargo build ${CARGO_PROFILE:+--profile ${CARGO_PROFILE}} --workspace --all-targe
 >&2 echo "Pre-building tests..."
 cargo test --no-run ${CARGO_PROFILE:+--profile ${CARGO_PROFILE}} --workspace --all-targets
 
+# We've just built everything there is to built, so we should not have a
+# need to be build things again from now on, but since cargo does not
+# let us enforce it, we need to go behind its back. We put a fake 'rustc'
+# in the PATH.
+# If you really need to break this rule, ping dpc
```

Comment:
Waiting for the movie: "Fearless band of novice developers embarks on epic journey to find the master programmer mentioned in the docs in hope for guidance. Will they be able to cross the rusty plains and scale the mountain of Nix?"